apps/server/src/modules/federal-state/controller/dto/federal-state.response.ts

```diff
@@ -17,7 +17,7 @@ export class FederalStateResponse {
 	}
 
 	@ApiProperty()
-	id: string;
+	_id: string;
```

Comment:
_id is specific to only mongoDB, and has type ObjectId. But mikroORM allows the use of a general id instead of type string, which should be used

```suggestion
@ApiProperty({
pattern: '[a-f0-9]{24}',
})
id: EntityId;
```